src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  position: relative;
+  height: 100px;
+  box-sizing: border-box;
```

Comment:
Use * selector only for box-sizing property

move height and position to &__day class